helm/provider.go

```diff
@@ -500,6 +500,13 @@ func (m *Meta) buildTunnel(d *schema.ResourceData) error {
 		return nil
 	}
 
+	// Wait a reasonable time for tiller, even if we didn't deploy it this run
+	o := &installer.Options{}
+	o.Namespace = d.Get("namespace").(string)
```

Comment:
Can we make this `m.Settings.TillerNamespace` to be consistent with the rest of the code in this function? 
```suggestion
	o.Namespace = m.Settings.TillerNamespace
```

Reply:
@meyskens I've updated with your suggestion. I'd used the same code as on line 437 for your reference. 
https://github.com/terraform-providers/terraform-provider-helm/blob/master/helm/provider.go#L437
Thanks for the review - hopefully you're able to approve. 😸